fabpolish/contrib.py

```diff
@@ -129,3 +129,9 @@ def check_image_edited():
 def composer_validate():
     info('Running composer validate...')
     return local('composer validate')
+
+
+@sniff(severity='major', timing='fast')
+def run_eslint():
+    info('Running ESLint...')
+    return local("./node_modules/eslint/bin/eslint.js .")
```

Comment:
`git ls-files | grep '\.js$' | xargs ./node_modules/eslint/bin/eslint.js` will run the lint on only versioned javascript files.